ui/raidboss/data/04-sb/raid/o4n.txt

```diff
@@ -0,0 +1,98 @@
+# DELTASCAPE V4.0--O4N
+# EXDEATH
+
+# Enrage appears to be 25 minutes. No way to accurately warn about it
+# as the second phase is HP-dependent.
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# -ii 24BA 24BC 24BE 24C3 24C5
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start"
+2.5 "--sync--" sync /:Exdeath:366:/ window 2.5,1
+9.5 "Doom" sync /:Exdeath:24B7:/ window 9.5,5
+19.1 "Blizzard/Fire/Thunder III" sync /:Exdeath:24B[9BD]:/
+28.8 "Blizzard/Fire/Thunder III" sync /:Exdeath:24B[9BD]:/
+37.4 "Blizzard/Fire/Thunder III" sync /:Exdeath:24B[9BD]:/
+50.3 "Mega Blizzard/Fire/Thunder" sync /:Exdeath:24(BF|C[01]):/
+63.6 "Mega Blizzard/Fire/Thunder" sync /:Exdeath:24(BF|C[01]):/
+74.8 "Mega Blizzard/Fire/Thunder" sync /:Exdeath:24(BF|C[01]):/
+87.0 "Mega Blizzard/Fire/Thunder" sync /:Exdeath:24(BF|C[01]):/
+92.6 "Blizzard/Fire III" sync /:Exdeath:24B[9B]:/
+103.2 "Thunder III (Buster)" sync /:Exdeath:24BD:/
+
+113.3 "Vacuum Wave" sync /:Exdeath:24B8:/ window 40,30
+121.5 "Holy" sync /:Exdeath:24C4:/
+132.8 "Mega Blizzard/Fire/Thunder" sync /:Exdeath:24(BF|C[01]):/
+138.4 "Blizzard/Fire III" sync /:Exdeath:24B[9B]:/
+149.1 "Thunder III (Buster)" sync /:Exdeath:24BD:/
+
+159.2 "Vacuum Wave" sync /:Exdeath:24B8:/ window 20,20
+167.4 "Holy" sync /:Exdeath:24C4:/
+178.7 "Mega Blizzard/Fire/Thunder" sync /:Exdeath:24(BF|C[01]):/
+184.3 "Blizzard/Fire III" sync /:Exdeath:24B[9B]:/
+195.0 "Thunder III (Buster)" sync /:Exdeath:24BD:/ jump 149.1
+
+205.1 "Vacuum Wave"
+213.3 "Holy"
+224.6 "Mega Blizzard/Fire/Thunder"
+230.2 "Blizzard/Fire III"
+
+# Phase change appears to be 65% HP.
+# This first section is not quite the same as the main rotation.
+300.0 "The Decisive Battle" sync /:Exdeath:2408:/ window 300,5
+305.9 "--sync--" sync /:Exdeath:240A:/
+307.0 "Collision" sync /:Exdeath:24CA:/
+312.0 "Zombie Breath" sync /:Exdeath:24CB:/
+322.4 "Holy" sync /:Exdeath:24C4:/ window 20,20
+329.5 "Flare" sync /:Exdeath:24C2:/
+338.6 "Meteor" sync /:Exdeath:24C6:/
+353.8 "Black Hole" sync /:Exdeath:24C8:/ window 30,30
+365.1 "Mega Blizzard/Fire/Thunder" sync /:Exdeath:24(BF|C[01]):/
+
+# This section has choices.
+370.7 "Blizzard/Fire III?" sync /:Exdeath:24B[9B]:/
+372.2 "Holy?" sync /:Exdeath:24C4:/
```

Comment:
Are there random holies? Just curious what this looks like.  Is it always blizzard + holy, or how does the flare later relate to this?

Reply:
As best I can tell, it's Holy OR one of the two elemental spells. No difference in the timing of the rest of the block, just those two things. I thought it might be "one gets paired with Flare, one with Vacuum Wave", but I've seen blocks with all four different combinations.

Reply:
Huh, ok! Mostly just curious.  There's not a lot of documentation about o4n out there, so I figured I might as well ask while it was fresh.

Reply:
I suppose I might as well ask here, did you or the people farming logs for you ever discover enrages on Eden's Gate/Verse normals?

Reply:
I don't think I ever did.  I think most everything from Stormblood onward came from logs at the time, and duty finder doesn't usually get there, if it even existed.